docs/migrating-v1alpha1-to-v1beta1.md

```diff
@@ -60,11 +59,154 @@ spec:
     value: https://example.com/foo.json
 ```
 
-#### Input and Output PipelineResources
+## PipelineResources and Catalog Tasks
+
+`PipelineResources` are remaining in alpha while the other resource kinds are
+promoted to beta. This is part of a deliberate effort to migrate users away
+from `PipelineResources` onto a broader suite of reusable `Tasks` provided by
+the Tekton catalog.
+
+_More on the reasoning and what's left to do in
+[Why aren't PipelineResources in Beta?](docs/resources.md#why-arent-pipelineresources-in-beta)._
+
+In the near-term we plan to continue to merge fixes to existing `PipelineResource` types, but
+are pushing back against adding significant new features or new `PipelineResource` types, and
+encourage people to use `Tasks` for those instead. At some point the feature may well be deprecated
+and subsequently replaced.
```

Comment:
I continue to disagree with this general statement, I still believe that pipeline resources are a much stronger abstraction than Tasks. I think we should fix / re-design the implementation behind pipeline resources and keep the abstraction, not walk away from it.

Reply:
Maybe we could change the language here to `At some point the feature in its current form could be redesigned, replaced, deprecated, or removed entirely`?

Reply:
Okay I've updated this, let me know if the new language is still too strong @afrittoli